### PR TITLE
WFCORE-5765 Unable to check the result containing whitespace with the…

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/WFCORE-5765

### CODE REVIEW

**Comment by jmesnil** [comment]
@jfdenise Could you please review?

File: cli/src/main/java/org/jboss/as/cli/handlers/ifelse/ConditionArgument.java [hunk 1]
```diff
@@ -289,7 +289,7 @@ public void leavingState(ParsingContext ctx) throws CommandFormatException {
         @Override
         public void character(ParsingContext ctx) throws CommandFormatException {
             //System.out.println(ctx.getState().getId() + ": " + ctx.getCharacter());
-            if(parenthesisState != ctx.getState() && !Character.isWhitespace(ctx.getCharacter())) {
+            if(parenthesisState != ctx.getState() && (!Character.isWhitespace(ctx.getCharacter()) || ctx.getState().getId().equals(QuotesState.ID) || ctx.getState().getId().equals(EscapeCharacterState.ID))) {
```
Comment (jfdenise) [inline cli/src/main/java/org/jboss/as/cli/handlers/ifelse/ConditionArgument.java:292]: @RanabirChakraborty just a minor comment, could you cut this line after the first '||'. Thank-you.

Comment (RanabirChakraborty) [reply cli/src/main/java/org/jboss/as/cli/handlers/ifelse/ConditionArgument.java]: Thanks for the review @jfdenise, Did the changes.

**Review by jfdenise — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks, @RanabirChakraborty

**Pull request merged**